desktop/electron-builder.json

```diff
@@ -3,7 +3,7 @@
 	"appId": "com.automattic.wordpress",
 	"asar": false,
 	"directories": {
-		"buildResources": "./resource/icons",
+		"buildResources": "./resource",
```

Comment:
Why is this change necessary? Will it interfere with the way assets for the other Windows targets are located/generated?

Reply:
We need this because the appx folder with resources for that target needs to be exposed in the root buildResources folder.

I updated the reference path to the icon file so it will be used properly still.